### ISSUE TITLE
Error when starting

### ISSUE THREAD

**Issue by schmid-mat** [EnFlexIT/AgentWorkbench#17]
After extracting the archive and starting the .exe-file a error occurs:
![image](https://user-images.githubusercontent.com/130981181/232502221-ccae418b-c12e-4e03-8a26-cf717d1689f2.png)

**Comment by CDerksen** [comment]
I guess the storage location of Agent.Workbench (AWB) is the problem:
Assuming that you have not executed AWB as an Administrator, I guess it cannot create further sub directories, nor extract further content or even write local property files to the current directory.
=> In that case: use a directory on your system, where the current user / AWB ist able to create directories and write files.

What does the content of the specified log file tells us? - Permission problems?

**Comment by schmid-mat** [comment]
The AWB does not have any permission problems. According to the log file it requires the bundle org.eclipse.equinox.common
Can you tell me how do I get this bundle. Unfortunately, I could not find something helpful on the Internet

[1681794866970.log](https://github.com/EnFlexIT/AgentWorkbench/files/11258118/1681794866970.log)

**Comment by CDerksen** [comment]
Ah, ok: having a look into the log file I recognized that you're using Java 8 instead of Java 11.
Eclipse already moved to Java 11 with Eclipse 4.17 (2020-09) - so we had to move as well.

If the remaining directory structure looks similar like shown in the image below, its most probably the mentioned Java version problem. BTW: Even Microsoft provides support to install JDK's!

![grafik](https://user-images.githubusercontent.com/29704527/232734257-5e761b60-d5a9-484f-aedf-99c847ca36e7.png)
